lib/tmuxinator/hooks.rb

```diff
@@ -0,0 +1,39 @@
+module Tmuxinator
+  module Hooks
+    def hook_on_reattach
```

Comment:
@endorama Thanks for clearing that up. Let's revisit the proposed refactoring if/when additional _Tmuxinator_ hooks are added.